_posts/2020-09-10-rustpython-usecase.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title:  "Use Case: RustPython Notebook"
+excerpt: "A use case for scientific computing in the browser."
+---
+
+**Disclaimer: opinions are my own. What you read in this post does not necessarily represent the views of the whole RustPython team -- Mireille**  
+
+The RustPython Notebook is a **toy** notebook inspired by the now inactive [Iodide project](https://alpha.iodide.io/). It is a testament to how powerful and inspiring the Iodide project was.  
+You can find the RustPython notebook code on [this link](https://github.com/RustPython/RustPython/tree/master/wasm/notebook) under `RustPython/wasm/notebook`. 
```

Comment:
Oh, right, we still don't have the notebook hosted on the website. I'll look into that.

Reply:
that would be cool to have it featured with or next to the one that you built.

Reply:
Can we release a new version of RustPython and notebook example on the website before publishing this article?